client/src/cordova/apple/xcode/ios/Outline.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -14,8 +14,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/getsentry/sentry-cocoa",
       "state" : {
-        "revision" : "cf43eac1aa12017868c257ad3854ad87a5de0758",
-        "version" : "7.31.5"
+        "revision" : "7fc7ca43967e2980d8691a8e017c118a84133aac",
+        "version" : "8.26.0"
```

Comment:
This needs to be applied to macOS too.

Reply:
I think we need to update this file for macos too?

Reply:
I'm not sure what this file is doing here actually. It seems to have been newly created in #1958. The ios and macos `Package.resolved` files are already correctly updated in `client/src/cordova/apple/ios.xcworkspace/xcshareddata/swiftpm/Package.resolved` and `client/src/cordova/apple/macos.xcworkspace/xcshareddata/swiftpm/Package.resolved` respectively. Removing this one.

Reply:
I think it works fine because we specify the full version and the revision is probably mapped 1:1 to the full version.

This picks the version if the version is not fully specified, similar to package-lock.json or go.sum